server/src/Persistence.hs

```diff
@@ -61,7 +60,7 @@ apply op val = do
     LBS8.hPutStrLn journalHandle entry
     for_ (pcMetrics . pvConfig $ val) $ \metrics -> do
       journalPos <- hTell journalHandle
-      Metrics.incrementJournalWritten (LBS8.length entry) metrics
+      _ <- Metrics.incrementJournalWritten (LBS8.length entry) metrics
```

Comment:
Can also be written as `void $ Metrics.increment...`

Why does it return a `Bool` though, what does it mean?

Reply:
Apparently it returns whether it actually increased the counter. When the argument is negative, it won't change the counter and return `False`, otherwise, it'll add the value and return `True`.

Reply:
I added a comment.